index/scorch/segment/zap/contentcoder.go

```diff
@@ -173,3 +188,18 @@ func (c *chunkedContentCoder) Write(w io.Writer) (int, error) {
 	}
 	return tw, nil
 }
+
+// ReadDocValueBoundary elicits the start, end offsets from a
+// starting offset based metaData header slice
+func ReadDocValueBoundary(chunk int, metaHeaders []MetaData) (uint64, uint64) {
+	var start uint64
+	if chunk > len(metaHeaders) {
```

Comment:
wondering if you need this case?  The callers seem to always check that chunk is < len(metaHeaders)?

Reply:
since it is an exported method, I felt we need to check this..?